MeasureAuthoringTool/mat-drupal
# External Link Disclaimer

## Description
Adds and configures `extlink` module as a dependency. Provides a modification to that module which uses a magnific popup dialog box rather than the browser's confirm functionality.

## To Test
1. composer install
2. import config
3. build pattern lab
4. clear cache
5. Load up the Drupal site, and try clicking on the social links at the bottom. You should see the external link disclaimer dialog box pop up.
6. Click no to close the box.
7. Try again, and click yes to open the external link in a new tab.
8. See that the dialog box is also closed when you navigate back to the original tab.

<img width="863" alt="Screen Shot 2020-02-25 at 12 21 55 PM" src="https://user-images.githubusercontent.com/1919263/75275013-6f906b00-57c9-11ea-944a-953d86a5cc34.png">
